### PR TITLE
Story redux

### PR DESCRIPTION
**Pull request by EdikoCruz**
Create feed component
Add stories to redux
Get stories from redux in feed
Add story from storyInput to redux

### CODE REVIEW

File: src/components/Feed/Feed.js [hunk 1]
```diff
@@ -13,12 +13,13 @@ import s from './Feed.scss'
...
 import {connect} from 'react-redux'
+import {addStory} from '../../actions/stories'
 
-function Feed ({stories}) {
+function Feed ({stories,addStory}) {
   return (
     <div className={ s.root }>
       <div className={ s.container }>
-        <StoryInput />
+        <StoryInput onClick={ addStory } />
```
Comment (cassioscabral) [inline src/components/Feed/Feed.js]: The StoryInput should be outside of Feed, but within Home

File: src/components/StoryInput/StoryInput.js [hunk 2]
```diff
@@ -18,19 +18,29 @@ const messages = defineMessages({
...
+class StoryInput extends Component {
+  render (){
+    return (
+      <div className={ s.root } >
+        <div className={ s.container } >
+          <textarea
+            rows="3"
+            placeholder={ messages.inputPlaceholder.defaultMessage }
+            className={ cn(su.noOutline, su.noResize) }
+            ref="input"
```
Comment (cassioscabral) [inline src/components/StoryInput/StoryInput.js]: Change to a better name than input, should be more meaningful

File: src/components/StoryInput/StoryInput.js [hunk 3]
```diff
@@ -18,19 +18,29 @@ const messages = defineMessages({
...
+          />
+          <button onClick={ () => {
+              if (this.refs.input.value !== '') {
+                this.props.onClick(this.refs.input.value)
```
Comment (cassioscabral) [inline src/components/StoryInput/StoryInput.js]: Reuse the this.refs.input.value using a variable

File: src/components/StoryInput/StoryInput.js [hunk 4]
```diff
@@ -18,19 +18,29 @@ const messages = defineMessages({
...
-        />
-        <button><FormattedMessage { ...messages.button }  /></button>
+class StoryInput extends Component {
+  render (){
+    return (
+      <div className={ s.root } >
+        <div className={ s.container } >
+          <textarea
+            rows="3"
+            placeholder={ messages.inputPlaceholder.defaultMessage }
```
Comment (cassioscabral) [inline src/components/StoryInput/StoryInput.js]: inputPlaceholder is too broad, if we had to add more inputs and following this pattern we would always have to change the inputPlaceholder to something like storyInputPlaceholder.

File: src/components/Feed/Feed.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+import StoryInput from '../StoryInput'
+import {connect} from 'react-redux'
+
+function Feed ({stories}) {
+  return (
+    <div className={ s.root }>
+      <div className={ s.container }>
+        <StoryInput />
+        <ul className={ s.stories }>
+          { stories.map((item, index) => (
```
Comment (cassioscabral) [inline src/components/Feed/Feed.js]: Create a function that returns all the Stories. The function can be placed in this same file. Improves readability and separate concerns.

File: src/routes/home/Home.js [hunk 6]
```diff
@@ -1,46 +1,30 @@
...
-import StoryInput from '../../components/StoryInput'
+import {connect} from 'react-redux'
+import {loadStories} from '../../actions/stories'
 
 const title = 'Rate Issues'
 
-function Home ({stories}, context) {
+function Home ({stories,loadStories}, context) {
   context.setTitle(title)
+  loadStories({stories: stories})
```
Comment (cassioscabral) [inline src/routes/home/Home.js:14]: This should be changed in the future. To load the stories we should place somewhere associated with the component mount hook. This is a case where we will be dealing with cache like "has the stories changed ? if not, give me the cached stories"

File: src/data/queries/stories.js [hunk 7]
```diff
@@ -11,7 +11,19 @@ import {GraphQLList as List} from 'graphql'
...
+  {id:'07',body:'https://twitter.com/dan_abramov/status/738286536161136644',publishedDate: new Date()},
+  {id:'08',body:'08',publishedDate: new Date()},
+  {id:'09',body:'09',publishedDate: new Date()},
+  {id:'10',body:'10',publishedDate: new Date()},
+  {id:'11',body:'11',publishedDate: new Date()},
+  {id:'12',body:'12',publishedDate: new Date()},
+  {id:'13',body:'13',publishedDate: new Date()}
 ]
 
 const news = {
```
Comment (cassioscabral) [inline src/data/queries/stories.js]: Why news ? shouldn't be stories ?

File: src/data/queries/stories.js [hunk 8]
```diff
@@ -8,46 +8,15 @@
...
 
-// React.js News Feed (RSS)
-const url = 'http://ajax.googleapis.com/ajax/services/feed/load' +
-            '?v=1.0&num=10&q=https://reactjsnews.com/feed.xml'
-
-let items = []
-let lastFetchTask
-let lastFetchTime = new Date(1970, 0, 1)
+let items = [
+  {id:'1',body:'medium.com/javascript-scene/12-books-every-javascript-developer-should-read-9da76157fb3#.z2x12e8p6',publishedDate: new Date()}
```
Comment (cassioscabral) [inline src/data/queries/stories.js]: Where are the others seed data ?
